winspool/winspool.go

```diff
@@ -154,7 +154,7 @@ func convertPrinterState(wsStatus uint32) *cdd.PrinterStateSection {
 		}
 		state.VendorState.Item = append(state.VendorState.Item, vs)
 	}
-	if wsStatus&PRINTER_STATUS_OFFLINE != 0 {
+	if wsStatus&PRINTER_STATUS_OFFLINE != 0 || wsAttributes&PRINTER_ATTRIBUTE_WORK_OFFLINE != 0 {
```

Comment:
Apologies for assuming this was driver-specific. This perhaps explains why MS have taken PRINTER_ATTRIBUTE_WORK_OFFLINE out of the printer API docs, since Windows manages it internally. The only other (uninteresting) mention I found was this: https://support.microsoft.com/en-us/kb/202480